src/app/shared/libs/actions.lib.ts

```diff
@@ -0,0 +1,17 @@
+import { ActionCreator, TypedAction } from '@ngrx/store/src/models';
+import { ICorrelatedAction } from 'dist/ngrx-auto-entity/lib/actions';

```

Comment:
We should fix this import, so it pulls from `@briebug/ngrx-auto-entity`. This import probably ended up this way because `ICorrelatedAction` is not included in the libraries exports, which I have already created a change request for (See above).

Reply:
Should be fixed now, along with the `ICorrelatedAction` export.